Code/GraphMol/Wrap/rough_test.py

```diff
@@ -6142,7 +6142,20 @@ def testMolsFromPNG(self):
     for mol, refMol in zip(mols, refMols):
       self.assertEqual(Chem.MolToSmiles(mol), Chem.MolToSmiles(refMol))
 
-
+  def test_github3403(self):
+    core1 = "[$(C-!@[a])](=O)(Cl)"
+    sma = Chem.MolFromSmarts(core1)
```

Comment:
please use the unittest methods here instead of assert (like the rest of the tests)